requirements.txt

```diff
@@ -1,21 +1,4 @@
-astroid==2.3.2
-certifi==2019.9.11
-chardet==3.0.4
-idna==2.8
-isort==4.3.21
-lazy-object-proxy==1.4.2
-mccabe==0.6.1
-oauthlib==3.1.0
-PlexAPI==4.3.1
-pylint==2.4.3
+plexapi==4.4.1
 python-dotenv==0.10.3
```

Comment:
0.15.0

Reply:
Fixed.

Now I checked all four, they are the same in Pipenv and in requirements.txt

Probably should not do multiple things in parallel :D